beaglecli

```diff
@@ -41,6 +57,8 @@
   beagle_cli.py file-types list
   beagle_cli.py file-group create <file_group_name> <storage>
   beagle_cli.py file-group list [--page-size=<page_size>]
+  beagle_cli.py run list [--page-size=<page_size>] [--request-id=<request_id>]...
+  beagle_cli.py run [--run-id=<run_id>]
```

Comment:
Can you just make this be `beagle_cli.py run get [--run-id=<run_id>]`. Also if you can add run create command like this `run create --app <app> --inputs=<inputs_path> [--name=run_name]` in this PR that would be great!

Reply:
Oh and also I think `beagle_cli.py` in the `USAGE` string now needs to be changed to `beaglecli`.